### ISSUE TITLE
Simplify Flask documentation example

### ISSUE THREAD

**Issue by lccro** [prometheus/client_python#562]
The Flask example can be much simplified like this

```python
from flask import Flask
from werkzeug.middleware.dispatcher import DispatcherMiddleware
from prometheus_client import make_wsgi_app

# Create my app
app = Flask(__name__)

# Add prometheus wsgi middleware to route /metrics requests
app.wsgi_app = DispatcherMiddleware(app.wsgi_app, {
    '/metrics': make_wsgi_app()
})
```

It simply instruments your `app` object, no need to change semantics.

**Comment by lccro** [comment]
Use `app.wsgi_app`/`app.wsgi_app` instead of `app_dispatch`/`app` in case it wasn't clear.

**Comment by brian-brazil** [comment]
I've never used Flask so I'll have to take your word for it. Would you like to send a PR?

**Comment by dgzlopes** [comment]
I think this can be closed! #563 was merged.

**Comment by lccro** [comment]
Of course! Thanks.
